### PR TITLE
adding plotFlowConc

### PR DESCRIPTION
**Pull request by fawda123**
If interested, I've added a function that produces a plot similar to Fig 8 in Beck and Hagy 2015 that shows variation in the conc/flow relationship by year.  Note that I've added some package dependencies.  Happy to make any changes as needed.

Also, I'm not sure how to backtransform to linear scale on the y-axis using the interpolation surfaces.  This would be a nice feature to include in a later commit, e.g., an argument for `logscale = FALSE`.

Hope this helps.  Bob was asking about this the other day.  

-Marcus

### CODE REVIEW

File: EGRET.Rproj [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Version: 1.0
```
Comment (ldecicco-USGS) [inline EGRET.Rproj]: No matter what, I'd remove this file from the pull request. If you want, you could add it to the gitignore.

**Comment by ldecicco-USGS** [comment]
Gah, I'm very torn. I've actively tried to minimize adding new dependencies (there are instances of EGRET being put on parallel computing clusters, and adding new dependencies is a mild pain. But I love the graph, and I use dplyr + tidyr + ggplot2 all the time...

Let me think about it a bit....

**Comment by fawda123** [comment]
Yea I figured you were excluding those packages for good reason.  They do come with a lot of overhead.... http://www.r-bloggers.com/evading-the-quothadley-taxquot-faster-travis-tests-for-r/

This could be redone using base packages but not a trivial effort.  I can look into this if you really do not want to add the dependencies.

**Comment by ldecicco-USGS** [comment]
Another option might be to make WRTDS it's own package that EGRET calls. Then all the complex models being run on "sandboxed" clusters don't need to care so much if the EGRET package starts to expand the dependency list.

**Comment by rmhirsch49** [comment]
Just to weigh in here.  I'm really just interested for now in giving this a
try and not expanding the EGRET package.  It seems to me that about the
only thing that is really needed for doing these figures is the lattice
package.  My interest was just to see the code and then see if we could
modify it to our liking.  No need to do more Marcus, unless you really want
to.  Laura and I can talk about how we might proceed.

I do like these plots and see this among a set of possible enhancements
that we might do in some future release of additional functionality.

Bob

On Wed, Jun 15, 2016 at 12:25 PM, Laura DeCicco <notifications@github.com>
wrote:

> Another option might be to make WRTDS it's own package that EGRET calls.
> Then all the complex models being run on "sandboxed" clusters don't need to
> care so much if the EGRET package starts to expand the dependency list.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/USGS-R/EGRET/pull/87#issuecomment-226241923>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/ABHd0eTK87wH-K6qSqUMPBk7FZ_v3wkwks5qMCeUgaJpZM4I2ahw>
> .
>



-- 

************************************************************
* Robert M. Hirsch
* Research Hydrologist
* 432 National Center                     rhirsch@usgs.gov
* U.S. Geological Survey                ph  703-648-5888
* Reston, VA 20192         http://profile.usgs.gov/rhirsch
*************************************************************

**Comment by ldecicco-USGS** [comment]
We're in the last push of getting our blog posts set up here...this might actually make a GREAT blog post.

**Comment by rmhirsch49** [comment]
Laura:  I agree, but we need to have code that you and I both have tested
and which uses a minimal number of other packages.  Do you want to try to
write that?

I will try to send you some other possible blog post items when you can
give me a feel for the format.

Bob

On Wed, Jun 15, 2016 at 2:51 PM, Laura DeCicco <notifications@github.com>
wrote:

> We're in the last push of getting our blog posts set up here...this might
> actually make a GREAT blog post.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/USGS-R/EGRET/pull/87#issuecomment-226283967>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/ABHd0YPDlSpMdtiA-mJPtRBp5WgBaGldks5qMEmggaJpZM4I2ahw>
> .
>



-- 

************************************************************
* Robert M. Hirsch
* Research Hydrologist
* 432 National Center                     rhirsch@usgs.gov
* U.S. Geological Survey                ph  703-648-5888
* Reston, VA 20192         http://profile.usgs.gov/rhirsch
*************************************************************

**Comment by fawda123** [comment]
Okay, that sounds fine.  I'll close the pull request for now until we get a better solution with the dependencies.  It's doable but would take a bit of effort.  At least you've got some examples to work with.  Thanks.

**Pull request closed without merging**